frontend/src/views/Transfers/AddEditViewTransfer.jsx

```diff
@@ -444,6 +445,22 @@ export const AddEditViewTransfer = () => {
                 </>
               )}
 
+            {/* Internal Comments */}
+            {!editorMode && (
+              <>
```

Comment:
suggestion: I don't think you need the wrapper component <> since you are returning a single component.

Reply:
Good catch! Removed the unnecessary `<></>` wrapper. Thanks!